meta-lmp-base/recipes-support/fioconfig/fioconfig/fioconfig.service

```diff
@@ -4,10 +4,11 @@ After=network.target
 ConditionPathExists=/var/sota/sota.toml
 
 [Service]
+EnvironmentFile=/etc/default/fioconfig
 RestartSec=10
 Restart=always
 ExecStartPre=mkdir -p /var/run/secrets
-ExecStart=/usr/bin/fioconfig daemon
+ExecStart=/usr/bin/fioconfig daemon -i ${FIOCONFIG_INTERVAL}
```

Comment:
You can just set the variable `DAEMON_INTERVAL` and not need this:

https://github.com/foundriesio/fioconfig/blob/481fe3ee2b595d7e1c3e10b98169ab5bb7befa46/main.go#L119

Reply:
With DAEMON_INTERVAL you won't need to change ExecStart.

Also, don't ship with a default configuration, this should instead be provided by a factory only (for cases where the default value is not enough).

Change EnvironmentFile to 

`EnvironmentFile=-/etc/default/fioconfig
`